express/parsers/apps/espresso/formats/txt.py

```diff
@@ -370,7 +379,7 @@ def _extract_data_from_bfgs_blocks(self, text, func):
             results.append(func(block))
         return results
 
-    def _extract_lattice(self, text, last_value=False, regex="lattice"):
+    def _extract_lattice(self, text, regex="lattice"):
```

Comment:
Same as above - which units is the lattice extracted in - angstrom?

Reply:
Done.